tests/testutils/iputils.go

```diff
@@ -82,9 +82,35 @@ func GetIPsv6(iface string) ([]string, error) {
 	return GetIPs(iface, netlink.FAMILY_V6)
 }
 
-func GetGW(iface string, family int) (string, error) {
-	return "", fmt.Errorf("Not implemented")
+func GetGW(iface string, Family int) (string, error) {
+	ifaces, err := net.Interfaces()
+	if err != nil {
+		return "", err
+	}
+
+	found := false
+	for _, ifa := range ifaces {
+		if ifa.Name != iface {
+			continue
+		}
+		found = true
```

Comment:
Is it necessary to iterate on the interfaces here? LinkByName() below should already check if it exists, and will return the error "Link %s not found" appropriately.